app/controllers/recipes_controller.rb

```diff
@@ -25,6 +25,16 @@ def create
     redirect_to recipes_path, alert: exception.message
   end
 
+  def destroy
+    @recipe = Recipe.find(params[:id])
```

Comment:
Could we make sure recipe belongs to current user? 
(Same suggestion I did for show method on previous PR)

Reply:
Great suggestion! I will add this to check if the recipe belongs to the current user. 
Thank you!